modules/data.atmosphere/R/met.process.R

```diff
@@ -25,7 +25,7 @@ met.process <- function(site, input, start_date, end_date, model, host, bety, di
   dbparms <- list(driver=driver, user=user, dbname=dbname, password=password, host=bety.host)
   con       <- db.open(dbparms)
   
-  met <- input$met$type
+  met <- input$met$met.source
```

Comment:
If the tag is <source> then this will be input$met$source not input$met$met.source